policies/cloud-sql-databases-instance-requires-all-incoming-connections-to-use-ssl/cloud-sql-databases-instance-requires-all-incoming-connections-to-use-ssl.sentinel

```diff
@@ -24,7 +24,12 @@ ip_configuration_require_ssl_is_true = rule when deny_undefined_ip_configuration
 	} as _, instances {
 		all instances.change.after.settings as _, setting {
 			all setting.ip_configuration as _, config {
-				config.require_ssl is true
+				(
+				# Check if require_ssl exists and is true
+				(config.require_ssl else null != null and config.require_ssl is true) or
+					# Check if ssl_mode exists and is not `ALLOW_UNENCRYPTED_AND_ENCRYPTED`
```

Comment:
It might be more robust to assert on the allowed values instead of excluding the disallowed one.

By explicitly asserting the allowed values, we ensure future changes to ssl_mode (like new modes being introduced) won’t inadvertently bypass this check.
This approach is also easier to understand at a glance since it defines the expected states directly.

Would you consider revising the check to something like:
```
config.ssl_mode in ["ENCRYPTED_ONLY", "TRUSTED_CLIENT_CERTIFICATE_REQUIRED"]
```
This way, the intent is immediately clear, and the logic is future-proofed :)